symex-custom.el

```diff
@@ -60,6 +60,10 @@
   :type 'list
   :group 'symex)
 
+(defcustom symex-common-lisp-backend 'slime
+  "Backend provider for Common Lisp interactive features; defaults to SLIME."
+  :type 'symbol
+  :group 'symex)
```

Comment:
nit: Would be good to mention the possible options here, slime or sly. [Here's an example docstring](https://github.com/greghendershott/racket-mode/blob/52e993f3b39416eeb4df6262491d4a42d0b35232/racket-custom.el#L210-L225) that we could follow. I realize this also affects the evil option above :)

Reply:
See https://github.com/countvajhula/symex.el/pull/44/commits/8982040871c9658e231a2b678358e305da34c6a8